### PR TITLE
add highlight to readme examples

### PR DESCRIPTION
**Pull request by thechampagne**

### CODE REVIEW

File: readme.markdown [hunk 1]
```diff
@@ -616,7 +616,7 @@ of the Nim compiler required is recommended.
 
 You can also specify multiple dependencies like so:
 
-```
+```ini
```
Comment (Araq) [inline readme.markdown:619]:
```suggestion
```nim
```

File: readme.markdown [hunk 2]
```diff
@@ -641,7 +641,7 @@ Both operators `^=` and `~=` were not available yet for Nimble 0.13.1 and
 earlier and would cause error messages if used there.
 Other more complex comparison operators that would be available in npm like
 `!=`, `||`, `-`, `*` and `X` are also not available in Nimble.
-```
+```ini
```
Comment (Araq) [inline readme.markdown:644]:
```suggestion
```nim
```

File: readme.markdown [hunk 3]
```diff
@@ -923,7 +923,7 @@ binary name.
 
 Dependencies are specified using the ``requires`` function. For example:
 
-```
+```ini
```
Comment (Araq) [inline readme.markdown:926]:
```suggestion
```nim
```

File: readme.markdown [hunk 4]
```diff
@@ -1024,7 +1024,7 @@ Use dot-separated numbers to represent the release version in the git
 tag label. Nimble will parse these git tag labels to know which
 versions of a package are published.
 
-``` text
+```sh
```
Comment (Araq) [inline readme.markdown:1027]:
```suggestion
``` text
```

Comment (thechampagne) [reply readme.markdown:1027]: text will not show comments

Comment (Araq) [reply readme.markdown:1027]: Well it's not shell code so I guess don't use any language tag then.

**Pull request merged**